website_and_docs/content/documentation/grid/getting_started.zh-cn.md

```diff
@@ -214,113 +203,83 @@ driver.quit();
 
 ## Querying Selenium Grid
 
-After starting a Grid, there are mainly two ways of querying its status, through the Grid 
-UI or via an API call.
+启动 Grid 后，主要有两种方式查询其状态，通过 Grid UI 或通过 API 调用。
 
-The Grid UI can be reached by opening your preferred browser and heading to 
-[http://localhost:4444](http://localhost:4444).
+可以通过打开您喜欢的浏览器并前往[http://localhost:4444](http://localhost:4444)。
 
-API calls can be done through the [http://localhost:4444/status](http://localhost:4444/status)
-endpoint or using [GraphQL]({{< ref "advanced_features/graphql_support.md" >}})
+API 调用可以通过 [http://localhost:4444/status](http://localhost:4444/status) 端点或使用 [GraphQL]({{< ref "advanced_features/graphql_support.md" >}})
 
 {{% pageinfo color="primary" %}}
-For simplicity, all command examples shown in this page assume that components are running
-locally. More detailed examples and usages can be found in the
-[Configuring Components]({{< ref "/configuration" >}}) section.
+为简单起见，本页中显示的所有命令示例均假定组件正在运行
+本地。 更详细的示例和用法可以在[配置组件]({{< ref "/configuration" >}}) 部分。
 {{% /pageinfo %}}
 
 ## Using the Java 11 HTTP Client {{% badge-version version="4.5" %}}
 
-By default, Grid will use [AsyncHttpClient](https://github.com/AsyncHttpClient/async-http-client). 
-AsyncHttpClient is an open-source library built on top of Netty. It allows the execution of HTTP 
-requests and responses asynchronously. Additionally it also provides WebSocket support. Hence it 
-is a good fit. 
+默认情况下，Grid 将使用 [AsyncHttpClient](https://github.com/AsyncHttpClient/async-http-client)。 AsyncHttpClient 是一个建立在 Netty 之上的开源库。 它允许异步执行 HTTP 请求和响应。 此外，它还提供 WebSocket 支持。 因此它很合适。
 
-However, AsyncHttpClient is not been actively maintained since June 2021. It coincides with the 
-fact that Java 11+ provides a built-in HTTP and WebSocket client. Currently, Selenium 
-has plans to upgrade the minimum version supported to Java 11. However, it is a sizeable effort. 
-Aligning it with major releases and accompanied announcements  is crucial to ensure the user 
-experience is intact.
+然而，AsyncHttpClient 从 2021 年 6 月开始就没有主动维护了。恰逢 Java 11+ 提供了内置的 HTTP 和 WebSocket 客户端。 目前，Selenium 计划将支持的最低版本升级到 Java 11。但是，这是一个相当大的努力。 将其与主要版本和随附的公告保持一致对于确保用户体验完好无损至关重要。
```

Comment:
sizeable effort? I think that means we have to do many works, not means try much?

Reply:
fixed